src/pages/validator/amplifier/integrator-onboarding.mdx

```diff
@@ -0,0 +1,86 @@
+# Integrate your chain
+
+## Prerequisites
+
+- You should know how to deploy and interact with a CosmWasm contract. Refer to the [Osmosis CosmWasm testnet deployment tutorial](https://docs.osmosis.zone/cosmwasm/testnet/cosmwasm-deployment/) if you’d like a refresher — just replace `osmosisid` with `axelard` and use the `arm64` version of the [contract optimizer](https://github.com/CosmWasm/optimizer) if you are working on a [Mac with Apple silicon](https://support.apple.com/en-us/116943). Run `--gas auto --gas-adjustment 2` to provide your transactions with sufficient gas.
+
+## Devnet information
+
+- Devnet RPC: `http://devnet.rpc.axelar.dev:26657`
+- Devnet LCD (REST API/JSON RPC): `http://devnet.rpc.axelar.dev:1317`
+- Devnet gRPC: `http://devnet.rpc.axelar.dev:9090`
+- Chain ID: `devnet-amplifier`
+- Other info can be found in [`devnet-amplifier.json`](https://github.com/axelarnetwork/axelar-contract-deployments/blob/e6f549de873e0ad211a073218f172d03f6e92c3b/axelar-chains-config/info/devnet-amplifier.json).
+
+## Set up your CosmWasm devnet environment
+
+1. Download the pre-built [`axelard v0.35.5` binary](https://github.com/axelarnetwork/axelar-core/releases/tag/v0.35.5) from GitHub.
+1. Generate an account by running `./bin/axelard keys add test`.
+1. Fund your devnet account with `uamplifier` tokens:
+    - Join the [Axelar Discord](https://discord.com/invite/axelar).
+    - Get the Developer Role.
+    - Go to the **#faucet** channel and submit a request with your account to get 100 test tokens. A sample request: `!faucet devnet axelar1suw3fsckht00c5srgtr8p6lhn58e9lxq9yf5th`
+1. Query your account balance by running `axelard q bank balances axelar1suw3fsckht00c5srgtr8p6lhn58e9lxq9yf5th —-node [RPC]`.
+1. Deploy your CosmWasm contract.
+
+## Run `ampd` / `tofnd`
+
+See the [Become an Amplifier Verifier (Worker)](/validator/amplifier/verifier-onboarding) for detailed instructions on how to run ampd and tofnd on your machine.
+
+## Onboard a chain to Amplifier
+
+1. Build all of the deployed contracts, such as the router and service registry, from [`devnet-amplifier.json`](https://github.com/axelarnetwork/axelar-contract-deployments/blob/e6f549de873e0ad211a073218f172d03f6e92c3b/axelar-chains-config/info/devnet-amplifier.json) using the [Cosmwasm `rust-optimizer`](https://github.com/CosmWasm/optimizer). From the root of the repo, run:
+
+    ```bash
+    docker run --rm -v "$(pwd)":/code \
+    --mount type=volume,source="$(basename "$(pwd)")_cache",target=/target \
+    --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
+    cosmwasm/optimizer:0.15.0
+    ```
+
+1. Deploy the verifier:
+
+    ```bash
+    axelard tx wasm store artifacts/voting_verifier.wasm --node http://devnet-amplifier.axelar.dev:26657 --from deployer --chain-id devnet-amplifier --gas-prices 0.007uamplifier --gas auto --gas-adjustment 2
+    # note the code id in the output
+    gas estimate: 6153520
```

Comment:
this is stdout of the cmd, it looks confusing in the rendered doc. let's remove it?
```suggestion
```